src/components/APIDoc/Operation.tsx

```diff
@@ -46,6 +50,20 @@ export const Operation: React.FunctionComponent<OperationProps> = props => {
 const OperationContent: React.FunctionComponent<OperationProps> = ({verb, path, operation, document}) => {
   const parameters = (operation.parameters || []).map(p => deRef(p, document));
 
+  const reqData: RequestFormat = {
+    method: verb.toUpperCase(),
+    url: "http://example.com"+path,
+    httpVersion: "HTTP/1.1",
+    cookies: [],
+    headers: [{name: "Accept", value: "application/json"}],
```

Comment:
I'm unsure at what extent this library supports other Content-types, not every API expects a json - some expect nothing or text - we could what is the operation expecting - this could be done in a follow up as we are already asuming it's a json anyway.

Reply:
@josejulio The library doesn't really care about the contents of the header array as long as there is an object with a `name` and `value`. We can parse the header info from the schema and add it here in that follow-up PR that I was talking about.